frb/scripts/pz_dm.py

```diff
@@ -52,14 +54,26 @@ def main(pargs):
     # Redshift estimates
 
     # Load
-    sdict = prob_dmz.grab_repo_grid()
-    PDM_z = sdict['PDM_z']
+    # consider CHIME telescope
+    sdict = prob_dmz.grab_chime_repo_grid() 
```

Comment:
this needs to be an `if, elif` with all the telescope options

Reply:
done